README.md

```diff
@@ -11,10 +11,10 @@ The `microsoft.excel` is a [Ballerina](https://ballerina.io/) connector for Micr
 
 This connector provides operations for connecting and interacting with Microsoft Graph API Excel endpoints over the network. 
 For more information about configuration and operations, go to the module. 
-- [`microsoft.excel`](excel/Module.md)
+- [`ballerinax/microsoft.excel`](excel/Module.md)
 
-# Building from the Source
-## Setting Up the Prerequisites
+# Building from the source
```

Comment:
Heading level should be ##

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/pull/15/commits/3e16c3fcae95051a7c24ecef4a633a6e7f3e29ef